app/workers/asn_logic_worker.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class AsnLogicWorker
+  include Sidekiq::Worker
+  include AsnLogic
+
+  def perform(args = {})
+    payload = {
+      network: args['network']
+    }
+
+    Pipeline.new(200, payload)
```

Comment:
Same code is in the demo file, maybe it's worth to move it to gather_asns_service to not duplicate and have it reusable?